lib/zkevm_bridge.star

```diff
@@ -61,3 +77,33 @@ def start_bridge_ui(plan, args, config):
             cmd=["run"],
         ),
     )
+
+    # Start the bridge UI gateway.
```

Comment:
This is the only change made in this PR: deploy nginx in front of the bridge UI.
The rest are cosmetic changes.